usaspending_api/common/query_with_filters.py

```diff
@@ -21,7 +21,9 @@ def generate_elasticsearch_query(cls, filter_values: List[str], query_type: _Que
         keyword_queries = []
         fields = [
```

Comment:
I think this is missing a few things from the list of fields shown here: https://usaspending-help.zendesk.com/hc/en-us/articles/360001255774-Keyword-Search-Question-How-does-the-Keyword-Search-work-